src/Components/NavbarHome/NavbarHome.js

```diff
@@ -105,13 +105,13 @@ function NavbarHome() {
               >
                 Ideas
               </Link>
-              <Link
+              {/* <Link
```

Comment:
Remove comments if not neccessary.

Reply:
this comment is necessary, the route exists, but page is not ready so as soon as the page is ready, ithu uncomment chethal mathi